Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115_test.py

```diff
@@ -347,3 +350,323 @@ def test_paging_command(mocker):
+    """
+    paging_data = util_load_json('test_data/command_results.json')
+    res = get_mapping_fields_command()
+
+    assert paging_data['get_mapping_fields'] == res.extract_mapping()
+
+
+@pytest.mark.parametrize(
+    'xsoar_status, xsoar_close_reason, expected_rsa_status',
+    [
+        (1, None, "New"),
+        (2, None, "Closed"),
+        (2, "False positive", "ClosedFalsePositive"),
+    ]
+)
+def test_xsoar_status_to_rsa_status(mocker, xsoar_status: int, xsoar_close_reason: str | None, expected_rsa_status: str):
+    """
+    Given:
+        -  XSOAR Status int
+        -  XSOAR Close reason
+    When
+        - running xsoar_status_to_rsa_status
+    Then
+        - Return RSA Status
+    """
+    assert xsoar_status_to_rsa_status(xsoar_status, xsoar_close_reason) == expected_rsa_status
+
+
+def test_update_remote_system_command_with_updated_incident(mocker):
+    """
+        Given:
+        - client with fetch parameters
+        - args with incident ID, status, delta_keys
+        - param no used by integration
+
+        When:
+            Calling the update_remote_system_command.
+
+        Then:
+            Check if RSA status has been updated.
+    """
+    paging_data = "INC-1"
+
+    class UpdateRemoteSystemArgsResponse:
+        def __init__(self) -> dict:
+            self.delta = {"key": "value"}
+            self.remote_incident_id = "INC-1"
+            self.data = {"status": 2, "closeReason": "False positive"}
+
+    mocker.patch.object(RSANetWitnessv115, "UpdateRemoteSystemArgs", return_value=UpdateRemoteSystemArgsResponse())
+    mocker.patch.object(client, "get_incident_request", return_value={"id": "INC-1", "status": "New"})
+    mocker.patch.object(client, "update_incident_request", return_value={"id": "INC-1", "status": "ClosedFalsePositive"})
+
+    assert update_remote_system_command(client, {}, {}) == paging_data
+
+
+def test_update_remote_system_command_with_nonupdated_incident(mocker):
+    """
+        Given:
+        - client with fetch parameters
+        - args with incident ID, status, delta_keys
+        - param no used by integration
+
+        When:
+            Calling the update_remote_system_command.
+
+        Then:
+            Check if RSA status has been updated.
+    """
+    paging_data = "INC-1"
+
+    class UpdateRemoteSystemArgsResponse:
+        def __init__(self) -> dict:
+            self.delta = {"key": "value"}
+            self.remote_incident_id = "INC-1"
+            self.data = {"status": 1, "closeReason": "New"}
+
+    mocker.patch.object(RSANetWitnessv115, "UpdateRemoteSystemArgs", return_value=UpdateRemoteSystemArgsResponse())
+    mocker.patch.object(client, "get_incident_request", return_value={"id": "INC-1", "status": "New"})
+    mocker.patch.object(client, "update_incident_request", return_value={"id": "INC-1", "status": "New", "assignee": None})
+
+    assert update_remote_system_command(client, {}, {}) == paging_data
```

Comment:
```suggestion
    mocker.patch.object(RSANetWitnessv115, "UpdateRemoteSystemArgs", return_value=UpdateRemoteSystemArgsResponse())
    mocker.patch.object(client, "get_incident_request", return_value={"id": "INC-1", "status": "New"})
    mocker_update_remote_system = mocker.patch.object(client, "update_incident_request")

    assert not(mocker_update_remote_system.called())
```
Please assert that the status wasn't updated by checking that the function wasn't called. Because it will always return the ID no matter what happens.